content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,150 @@
+
+    {{< /note >}}
+
+1. Update the kubelet configuration to use new CA on all worker nodes.
+
+1. Use the new CA to sign new certificates that replace the existing `apiserver.crt`, `apiserver-kubelet-client.crt`, and `front-proxy-client.crt` and place the new certificates in the Kubernetes certificates directory. You can use the existing private keys or new private keys. if you changed the private keys then update these in the Kubernetes certificates directory as well.
+
+    To generate certificates and private keys for your cluster using the `openssl` command line tool, see [Certificates (`openssl`)](/docs/concepts/cluster-administration/certificates/#openssl).
+    
+    You can also use [`cfssl`](/docs/concepts/cluster-administration/certificates/#cfssl).
+
+1. If your cluster is using bootstrap tokens to join nodes, update the ConfigMap `cluster-info` in the `kube-public` namespace.
+
+    ```shell
+    base64_encoded_ca="$(base64 /etc/kubernetes/pki/ca.crt)"
+
+    kubectl get cm/cluster-info --namespace kube-public -o yaml | \
+        /bin/sed "s/\(certificate-authority-data:\).*/\1 ${base64_encoded_ca}" | \
+        kubectl apply -f -
+    ```
+
+1. Update all service account tokens with the new CA certificates.
+    ```shell
+    base64_encoded_ca="$(base64 /etc/kubernetes/pki/ca.crt)"
+
+    for namespace in $(kubectl get ns --no-headers | awk '{print $1}'); do
+        for token in $(kubectl get secrets --namespace "$namespace" --field-selector type=kubernetes.io/service-account-token -o name); do
+            kubectl get $token --namespace "$namespace" -o yaml | \
+            /bin/sed "s/\(ca.crt:\).*/\1 ${base64_encoded_ca}" | \
+            kubectl apply -f -
+        done
+    done
+
+    ```
+
+1. Update certificates for user accounts.
+
+    For information about creating certificates for individual user accounts, see [Configure certificates for user accounts](/docs/setup/best-practices/certificates/#configure-certificates-for-user-accounts).
+
+    Update `client-certificate-data` and `client-key-data` respectively with the contents of certificates and private keys generated in their respective conf files, along with the kubelet client file in`/var/lib/kubelet/kubelet-client-current.pem`.
+
+1. Update `kubelet.conf`, the `kubelet` certificate, and the private key on all worker nodes as well.
+
+1. Update the `certificate-authority-data` section in all `admin.conf`, `controller-manager.conf`,  `scheduler.conf`, and `kubelet.conf` files with Base64-encoded CA certificate data:
+
+    Example:
+
+    ```shell
+    base64_encoded_ca="$(base64 /etc/kubernetes/pki/ca.crt)"
+
+    /bin/sed -i "s/\(certificate-authority-data:\).*/\1 ${base64_encoded_ca}" /path/to/conf/files
+    ```
+
+1. Follow these steps on all control plane and worker nodes.
+
+    1. Stop the kubelet service on all control plane and worker nodes.
+
+    1. Kill the containers manually on control plane nodes:
```

Comment:
Why is this necessary? Is this to restart all the control plane components with updated flags? If so, there isn't a need to stop all the containers on a node at the same time is there?